pkg/scripts/os_amzn.go

```diff
@@ -214,6 +226,8 @@ func KubeadmAmazonLinux(cluster *kubeoneapi.KubeOneCluster, force bool) (string,
 		"INSTALL_CONTAINERD":     cluster.ContainerRuntime.Containerd,
 		"USE_KUBERNETES_REPO":    cluster.AssetConfiguration.NodeBinaries.URL == "",
 		"CILIUM":                 ciliumCNI(cluster),
+		"USE_OBS":                kubeoneapi.IsOpenBuildServiceEnabled(),
+		"OBS_VERSION":            cluster.Versions.KubernetesMajorMinorVersion(),
```

Comment:
Theoretically we can get away with just 1 variable instead of 2, just "OBS_VERSION", and in template use it like this
```
{{ with .OBS_VERSION }}
  this block will be omitted if .OBS_VERSION above is empty. .OBS_VERSION from above here can be used as {{ . }}.
{{ end }}
```